### PR TITLE
update git clone link and supported install OS

### PR DESCRIPTION
**Pull request by laurafitzgerald**
### Summary

Small README fixes

### Change Type

Docs

### Check List Before Merging

- [ ] This PR passes all pre-commit hook validations.
- [ ] This PR is fully tested and regression tests are included.
- [ ] All CI checks and tests are passing.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ If you have not added `$GOPATH/bin` to your `$PATH`, then you may
 alternatively run `./mage build` and manually move the `bin/ocm-addons`
 binary to any other directory in your `$PATH`.
 
-### Option 3: Install using 'install.sh' (Not supported for Windows)
+### Option 3: Install using 'install.sh' (Not supported for Windows or Mac)
```
Comment (Ajpantuso) [inline README.md:41]: Can you paste the output from a failed run of the `install.sh` script on MacOS?

I'm assuming the arch type is not properly mapped for Apple Silicon, but if given the output I'll ask one of the Mac users on our team to work on a fix.

**Review by Ajpantuso — COMMENTED** [review]
/lgtm

**Review by Ajpantuso — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (laurafitzgerald) [reply README.md:41]: tar: Error opening archive: Unrecognized archive format

**Pull request merged**